Modules/Utility/os.lua

```diff
@@ -35,7 +41,7 @@ local function date(formatString, unix)
 			local UTC
 			formatString, UTC = gsub(formatString, "^!", "") -- If formatString begins in '!', use os.time()
 			assert(UTC == 0 or not unix, "Cannot determine time to format for os.date. Use either an \"!\" at the beginning of the string or pass a time parameter")
-			unix = UTC == 1 and os.time() or unix
+			unix = UTC == 1 and time() or unix
```

Comment:
I would prefer this at the top of the date function which helps clarify behavior of arguments if you're reading this code (this is pretty standard).

Thoughts